samples/client/petstore/php/SwaggerClient-php/lib/Model/MixedPropertiesAndAdditionalPropertiesClass.php

```diff
@@ -181,6 +181,11 @@ public function getUuid()
      */
     public function setUuid($uuid)
     {
+        if (is_null($uuid)) {
+            $this->container['uuid'] = $uuid;
+
+            return $this;
+        }
         $this->container['uuid'] = $uuid;
```

Comment:
@xaviapa I don't think I fully understand the change.

Without the "if" block that you added, `$this->container['uuid']` is still set to `$uuid`

What problem did you encounter when using the setter?